<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>谷哥番茄钟</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QToolButton" name="tool_btn_clock">
     <property name="font">
      <font>
       <family>Yuppy SC</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>番茄钟</string>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/Icons/pomo_button_2.png</normaloff>:/Icons/pomo_button_2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>44</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_round_progress_bar"/>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="slider_window_opacity">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tool_btn_settings">
           <property name="font">
            <font>
             <family>Yuppy SC</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>设置</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/Icons/settings_button.png</normaloff>:/Icons/settings_button.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="shortcut">
            <string>Alt+S</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tool_btn_start_pause">
           <property name="font">
            <font>
             <family>Yuppy SC</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/Icons/play_button_1.png</normaloff>:/Icons/play_button_1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="shortcut">
            <string>Space</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tool_btn_next">
           <property name="font">
            <font>
             <family>Yuppy SC</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>快进</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/Icons/fast-forward-button.png</normaloff>:/Icons/fast-forward-button.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="shortcut">
            <string>Right</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_pomo_working_time">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>番茄专注时间</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="MinutesSecondsBox" name="box_pomo_working_time_setting" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_pomo_rest_time">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>番茄休息时间</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="MinutesSecondsBox" name="box_pomo_rest_time_setting" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_pomo_long_rest_time">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>番茄长休息时间</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="MinutesSecondsBox" name="box_pomo_long_rest_time_setting" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_pomo_number">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>番茄钟个数</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="SpinSliderBox" name="box_pomo_number_setting" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_auto_start_working">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>番茄专注计时自动开始</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QGridLayout" name="gridLayout_auto_start_working">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <property name="spacing">
          <number>-1</number>
         </property>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_auto_start_break">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>番茄休息计时自动开始</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QGridLayout" name="gridLayout_auto_start_break">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <property name="spacing">
          <number>-1</number>
         </property>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_alarm_sound">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>闹铃声音


</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="SoundsControlPanel" name="alarm_sound_settings" native="true"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_ticking_sound">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>时钟声音

</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="SoundsControlPanel" name="ticking_sound_settings" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinSliderBox</class>
   <extends>QWidget</extends>
   <header location="global">Components/spinsliderbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MinutesSecondsBox</class>
   <extends>QWidget</extends>
   <header location="global">Components/minutessecondsbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SoundsControlPanel</class>
   <extends>QWidget</extends>
   <header location="global">Components/soundscontrolpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
